feat: log the transformer names and artifacts to a graph

HarikrishnanBalagopal opened the pull request:
Signed-off-by: Harikrishnan Balagopal <harikrishmenon@gmail.com>

<img width="1792" alt="image" src="https://user-images.githubusercontent.com/20921177/166171072-eaf47c21-b736-4af6-852a-c3a5ea8b0845.png">

ashokponkumar reviewed (changes requested):
Wow! Nice work @HarikrishnanBalagopal . Will greatly improve debuggability of Move2Kube. 

Few feature suggestions Hari. We can do it in another PR.

a. We should add this to Move2Kube ui too.
b. Would be good to make it a live graph, where the transform command continuously outputs the json, and the ui keeps refreshing.
c. If it is a live graph, we could show the current active node.
d. On hover, we can show the actual contents of the artifacts.

Couple of questions:
a. Why is readmegenerator and parameterizer listed on the top? Is it yet to be executed?
b. There seems to be repetitions of parameterizer and readmegenerator.
c. We are showing dependency in the same level as the iteration, which might be confusing. We have to check how we handle dependency and mandatory pass-through.

HarikrishnanBalagopal commented:
> Wow! Nice work @HarikrishnanBalagopal . Will greatly improve debuggability of Move2Kube.
> 
> Few feature suggestions Hari. We can do it in another PR.
> 
> a. We should add this to Move2Kube ui too. b. Would be good to make it a live graph, where the transform command continuously outputs the json, and the ui keeps refreshing. c. If it is a live graph, we could show the current active node. d. On hover, we can show the actual contents of the artifacts.
> 
> Couple of questions: a. Why is readmegenerator and parameterizer listed on the top? Is it yet to be executed? b. There seems to be repetitions of parameterizer and readmegenerator. c. We are showing dependency in the same level as the iteration, which might be confusing. We have to check how we handle dependency and mandatory pass-through.

- changed the block positions to match iterations (taking into account mandatory/on-demand passthrough)
- added path mappings
<img width="1792" alt="image" src="https://user-images.githubusercontent.com/20921177/166587074-df01f6cc-7068-4d39-bbf9-b27ff1547de0.png">

ashokponkumar reviewed (approved):
Nice work!
